### ISSUE TITLE
 uninitialized constant Chef::Resource::SumologicCollector on chef 13.1.31

### ISSUE THREAD

**Issue by potato20** [SumoLogic/sumologic-collector-chef-cookbook#131]
```
sumologic_collector '/opt/SumoCollector' do
  sumo_access_id  secrets['sumo_access_id']
  sumo_access_key secrets['sumo_access_key']
  ephemeral true
  clobber true
  skip_registration true
end
```

output

```
==> client-node-1: ================================================================================
==> client-node-1:     
==> client-node-1: Error executing action `install_and_configure` on resource 'sumologic_collector[/opt/SumoCollector]'
==> client-node-1:     
==> client-node-1: ================================================================================
==> client-node-1:     
==> client-node-1: 
==> client-node-1: 
==> client-node-1:     
==> client-node-1: NameError
==> client-node-1:     
==> client-node-1: ---------
==> client-node-1:     uninitialized constant Chef::Resource::SumologicCollector
==> client-node-1:     
==> client-node-1: 
==> client-node-1:     
==> client-node-1: Cookbook Trace:
==> client-node-1:     ---------------
==> client-node-1:     /var/chef/cache/cookbooks/sumologic-collector/providers/default.rb:9:in `load_current_resource'
==> client-node-1:     
==> client-node-1: 
==> client-node-1:     Resource Declaration:
==> client-node-1:     ---------------------
==> client-node-1:     
==> client-node-1: # In /var/chef/cache/cookbooks/letsrolloll/recipes/rofl.rb
==> client-node-1:     
==> client-node-1: 
==> client-node-1:     
==> client-node-1:  13: sumologic_collector '/opt/SumoCollector' do
==> client-node-1:     
==> client-node-1:  14:   sumo_access_id  secrets['sumo_access_id']
==> client-node-1:     
==> client-node-1:  15:   sumo_access_key secrets['sumo_access_key']
==> client-node-1:     
==> client-node-1:  16:   ephemeral true
==> client-node-1:     
==> client-node-1:  17:   clobber true
==> client-node-1:     
==> client-node-1:  18:   skip_registration true
==> client-node-1:      19: end
==> client-node-1:     
==> client-node-1:  20: 
==> client-node-1: 
==> client-node-1:     
==> client-node-1: 
==> client-node-1: 
==> client-node-1:     
==> client-node-1: Compiled Resource:
==> client-node-1: 
==> client-node-1:     ------------------
==> client-node-1:     
==> client-node-1: # Declared in /var/chef/cache/cookbooks/letsrolloll/recipes/rofl.rb:13:in `from_file'
==> client-node-1:     
==> client-node-1: 
==> client-node-1:     
==> client-node-1: sumologic_collector("/opt/SumoCollector") do
==> client-node-1:     
==> client-node-1:   action [:install_and_configure]
==> client-node-1:       default_guard_interpreter :default
==> client-node-1:     
==> client-node-1:   declared_type :sumologic_collector
==> client-node-1:     
==> client-node-1:   cookbook_name "letsrolloll"
==> client-node-1:     
==> client-node-1:   recipe_name "rofl"
==> client-node-1:     
==> client-node-1:   sumo_access_id "xxx"
==> client-node-1:     
==> client-node-1:   sumo_access_key "xxx"
==> client-node-1:     
==> client-node-1:   ephemeral true
==> client-node-1:     
==> client-node-1:   clobber true
==> client-node-1:     
==> client-node-1:   skip_registration true
==> client-node-1:     
==> client-node-1:   dir "/opt/SumoCollector"
==> client-node-1:     
==> client-node-1: end
==> client-node-1: 
==> client-node-1:     
==> client-node-1: 
==> client-node-1: 
==> client-node-1:     
==> client-node-1: System Info:
==> client-node-1: 
==> client-node-1:     
==> client-node-1: ------------
==> client-node-1:     
==> client-node-1: chef_version=13.1.31
==> client-node-1: 
==> client-node-1:     
==> client-node-1: platform=ubuntu
==> client-node-1:     
==> client-node-1: platform_version=16.04
==> client-node-1:     
==> client-node-1: ruby=ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
==> client-node-1:     
==> client-node-1: program_name=chef-client worker: ppid=11404;start=21:25:55;
==> client-node-1: 
==> client-node-1:     executable=/opt/chef/bin/chef-client
```

**Comment by RoboticCheese** [comment]
FWIW there's a fix for this in #122.

**Comment by potato20** [comment]
Ah, good to know. thanks!

**Comment by majormoses** [comment]
closed via #122
